src/js/themes/vanilla.js

```diff
@@ -284,6 +287,9 @@ export default deepFreeze({
     },
     size: `${baseSpacing}px`,
   },
+  rangeInput: {
+    trackColor: css`${props => rgba(props.theme.global.colors.text, 0.2)}`,
```

Comment:
Fine for now. I'm wondering if we'll eventually want to be able to do something like:
```
track {
  color: ...,
  width: ...,
}
```